pinot-broker/src/main/java/org/apache/pinot/broker/broker/helix/BaseBrokerStarter.java

```diff
@@ -346,9 +346,7 @@ public void start()
       _sqlQueryExecutor = new SqlQueryExecutor(_spectatorHelixManager);
     }
     LOGGER.info("Starting broker admin application on: {}", ListenerConfigUtil.toString(_listenerConfigs));
-    _brokerAdminApplication =
-        new BrokerAdminApiApplication(_routingManager, _brokerRequestHandler, _brokerMetrics, _brokerConf,
-            _sqlQueryExecutor, _serverRoutingStatsManager, _accessControlFactory, _spectatorHelixManager);
+    _brokerAdminApplication = createBrokerAdminApp();
     registerExtraComponents(_brokerAdminApplication);
```

Comment:
Let's move this call into default `createBrokerAdminApp()` as well

Reply:
Ah, typo.

Reply:
Done.

Reply:
Have you pushed the change?

Reply:
Done now.